src/40_Dynamic_AMP/Dynamic_Content.html

```diff
@@ -0,0 +1,45 @@
+<!-- ## Dynamic Content -->
```

Comment:
Can we find a better name for this? Dynamic content is very generic...

Reply:
Happy to change it if you can suggest a better name

Reply:
suggestion: Dynamic Content after User-Interaction